docs/aiohttp-icon.svg

```diff
@@ -1,487 +1 @@
-                    <circle id="Oval-111" opacity="0.8" fill="#FFFFFF" filter="url(#filter-12)" cx="170" cy="170" r="110"></circle>
-                    <circle id="Oval-112" fill="#F26262" filter="url(#filter-13)" cx="170" cy="170" r="81"></circle>
-                </g>
-            </g>
-            <mask id="mask-16" sketch:name="inner shadow" fill="white">
-                <use xlink:href="#path-15"></use>
-            </mask>
-            <use id="inner-shadow" fill="url(#radialGradient-14)" sketch:type="MSShapeGroup" xlink:href="#path-15"></use>
-        </g>
-    </g>
-</svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="363.82 73.82 75.36 75.36"><defs><style>.a{fill:#2c5bb4;}</style></defs><path class="a" d="M-70,37.679A37.74,37.74,0,0,1-32.32,0,37.74,37.74,0,0,1,5.359,37.679,37.741,37.741,0,0,1-32.32,75.36,37.741,37.741,0,0,1-70,37.679Zm37.68,34.553A34.377,34.377,0,0,0-10.936,64.8l-.952-2.061a2.343,2.343,0,0,1-.733.117,2.346,2.346,0,0,1-2.345-2.345,2.343,2.343,0,0,1,1.126-2l-2.266-4.909a3.905,3.905,0,0,1-1.2.189,3.91,3.91,0,0,1-3.644-2.492l-7.77,1.987q0,.018,0,.037A2.346,2.346,0,0,1-31.07,55.66a2.345,2.345,0,0,1-2.344-2.345,2.352,2.352,0,0,1,.065-.551l-10.906-4.689a3.906,3.906,0,0,1-3.388,1.957,3.933,3.933,0,0,1-.625-.05l-2.349,8.136a2.346,2.346,0,0,1,1.254,2.076,2.347,2.347,0,0,1-1.788,2.279l.164,4.268A34.336,34.336,0,0,0-32.32,72.232ZM-51.95,66.1l-.138-3.587a2.346,2.346,0,0,1-1.965-2.315,2.345,2.345,0,0,1,2.344-2.345q.1,0,.19.008l2.349-8.134a3.91,3.91,0,0,1-2.382-3.6v0l-14.676-1.778A34.651,34.651,0,0,0-51.95,66.1Zm41.765-1.91a34.8,34.8,0,0,0,8.6-10.741l-12.011-2.359A3.92,3.92,0,0,1-15.255,53.2l2.3,4.986a2.366,2.366,0,0,1,.333-.024,2.345,2.345,0,0,1,2.345,2.344,2.34,2.34,0,0,1-.8,1.76Zm9.036-11.612A34.334,34.334,0,0,0,2.232,37.679,34.384,34.384,0,0,0-5.593,15.811l-4.747,1.618a2.35,2.35,0,0,1,.065.551,2.345,2.345,0,0,1-2.345,2.345,2.346,2.346,0,0,1-2.142-1.389L-24.63,22.3a7.827,7.827,0,0,1,.283,2.091,7.81,7.81,0,0,1-3.47,6.5l8.969,15.391a3.9,3.9,0,0,1,1.537-.314A3.909,3.909,0,0,1-13.4,49.874q0,.146-.011.29Zm-27.778-.211,7.74-1.979a3.948,3.948,0,0,1-.033-.507,3.9,3.9,0,0,1,1.56-3.125l-8.967-15.387a7.783,7.783,0,0,1-3.538.844A7.781,7.781,0,0,1-36.3,31.026L-45,43.243a3.9,3.9,0,0,1,1.265,2.879,3.913,3.913,0,0,1-.154,1.088l10.937,4.7a2.341,2.341,0,0,1,1.88-.944A2.346,2.346,0,0,1-28.926,52.361Zm-22.513-7.172a3.911,3.911,0,0,1,3.8-2.975,3.891,3.891,0,0,1,1.882.482l8.7-12.213a7.8,7.8,0,0,1-2.92-6.093,7.787,7.787,0,0,1,1.936-5.15L-49.666,7.808A34.582,34.582,0,0,0-66.873,37.679a34.6,34.6,0,0,0,.47,5.7Zm26.5-23.778,9.969-3.4q0-.017,0-.034a2.346,2.346,0,0,1,2.345-2.345,2.342,2.342,0,0,1,1.878.94l4.5-1.534A34.485,34.485,0,0,0-32.32,3.127a34.325,34.325,0,0,0-16.5,4.2l11.43,11.244a7.787,7.787,0,0,1,5.222-2A7.82,7.82,0,0,1-24.934,21.411Z" transform="translate(433.82 73.82)"/></svg>
```

Comment:
Did it have to be one-liner?

Reply:
Sorry, I have no idea.
I did not edit the file manually :)

Reply:
It's much more usable in prettified form. Front-end dudes are able to work with that, so I'd keep it readable.